laterpay/application/Controller/Post.php

```diff
@@ -27,15 +27,15 @@ class LaterPay_Controller_Post extends LaterPay_Controller_Abstract
      */
     public function ajax_load_purchased_content() {
         if ( ! isset( $_GET['action'] ) || $_GET['action'] !== 'laterpay_post_load_purchased_content' ) {
-            exit;
+            wp_die();
```

Comment:
https://core.trac.wordpress.org/browser/tags/4.1.1/src/wp-includes/functions.php#L2361
By default this value is empty, so should be no errors here and Wordpress recommend to use this func instead of die() or exit().